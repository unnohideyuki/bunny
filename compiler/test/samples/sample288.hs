main = do print (pi :: Double)
          print (exp 1 :: Double)
          print (log 10 :: Double)
          print (sqrt 2 :: Double)
          print (pi ** 2 :: Double)
          print (logBase 2 10 :: Double)
          print (sin 1 :: Double)
          print (cos 1 :: Double)
          print (tan 1 :: Double)
          print (asin (-1) :: Double)
          print (acos (-1) :: Double)
          print (atan 1 :: Double)
          print (sinh 1 :: Double)
          print (cosh 1 :: Double)
          print (tanh 1 :: Double)
          print (asinh 1 :: Double)
          print (acosh 2 :: Double)
          print (atanh 0.5 :: Double)

