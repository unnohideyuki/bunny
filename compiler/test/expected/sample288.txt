3.141592653589793
2.718281828459045
2.302585092994046
1.414213562373095
9.869604401089358
3.3219280948873626
0.8414709848078965
0.5403023058681398
1.557407724654902
-1.5707963267948966
3.141592653589793
0.7853981633974483
1.1752011936438014
1.543080634815244
0.7615941559557647
0.8813735870195429
1.3169578969248166
0.5493061443340549
